config/routes.rb

```diff
@@ -18,6 +19,14 @@
       resources :settings, only: [] do
         get :must_update, on: :collection
       end
+      
+      # TOPICS CONTROLLER ROUTES
+      get :topics, to: 'topics#index'
+      get 'topics/show'
+      post :topics, to: 'topics#create'
+      get 'topics/update'
+      get 'topics/destroy'
```

Comment:
Rails has the prebuilt method `resources` that adds all the CRUD http routes to the controllers (index, show, new, create, edit, update, destroy):

`resources :topics`

then you can specify which ones you want with `only: [:index, :show, :create, :update, :destroy]`

So, on your case you may want to end up with:
`resources :topics, only: [:index, :show, :create, :update, :destroy]`